app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/BuyTicketActivity.java

```diff
@@ -0,0 +1,58 @@
+package de.tum.in.tumcampusapp.component.ui.ticket;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import java.text.DecimalFormat;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.component.other.generic.activity.BaseActivity;
+import de.tum.in.tumcampusapp.component.ui.ticket.model.Ticket;
+
+public class BuyTicketActivity extends BaseActivity {
+
+    public BuyTicketActivity() {
+        super(R.layout.activity_buy_ticket);
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        TextView eventView = (TextView) findViewById(R.id.ticket_details_event);
+        TextView locationView = (TextView) findViewById(R.id.ticket_details_location);
+        TextView dateView = (TextView) findViewById(R.id.ticket_details_date);
+        TextView priceView = (TextView) findViewById(R.id.ticket_details_price);
+        TextView ticketTypeView = (TextView) findViewById(R.id.ticket_details_ticket_type);
+        Button paymentButton = (Button) findViewById(R.id.paymentbutton);
+
+        // TODO: Get data from Api backend, now it is mock up data
+        Ticket ticket = TicketsController.getTickets();
+
+        String eventString = ticket.getEvent().getTitle();
+        String locationString = ticket.getEvent().getLocality();
+        String dateString = new SimpleDateFormat("yyyy-MM-dd hh:mm", Locale.GERMANY).
+                format(ticket.getEvent().getDate());
+        String priceString = new DecimalFormat("#.00").format(ticket.getType().getPrice())
+                + " €";
+        String ticketTypeString = ticket.getType().getDescription();
+
+        eventView.append(": " + eventString);
+        locationView.append(": " + locationString);
+        dateView.append(": " + dateString);
+        priceView.append(": " + priceString);
+        ticketTypeView.append(": " + ticketTypeString);
+
```

Comment:
I made some adjustments to the layout: mainly, I separated the ticket details view into views for event, location, date, price etc.